.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: node_js
 before_install:
-# See https://github.com/travis-ci/travis-ci/issues/5092#issuecomment-171777646
-# Dirty hack for https://github.com/travis-ci/travis-ci/issues/5092
-- |
-  PATH=`echo $PATH | sed "s/\.\/node_modules\/\.bin//g"`; export PATH=$PATH:./node_modules/.bin; touch .env; if [ $IS_INTEGRATION_TEST ]; then rvm install ruby-2.3.0 && rvm use ruby-2.3.0 && bundle install; else echo "Not an integration test. Skipping Ruby setup"; fi
+  touch .env;
+  if [ $IS_INTEGRATION_TEST ]; then rvm install ruby-2.4.3 --binary && rvm use ruby-2.4.3 && bundle install; else echo "Not an integration test. Skipping Ruby setup"; fi
```

Comment:
rvm should be able to pick up `2.4.3` from our .ruby-verison file. We could switch this to a `before_script` task to take advantage of that file.

Reply:
I did this, but it failed to find the `.ruby-version` file. Wonder if this gets run on a different path than the repo itself.